content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,92 @@
+
+## Telemetry
+
+- **Added** Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als) support for Envoy gRPC.
+- **Added** a Grafana dashboard for Citadel monitoring.
+- **Added** [metrics](/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+- **Added** control plane metrics to monitor Istio's configuration state.
+- **Added** telemetry reporting for traffic destined to the `Passthrough` and `BlackHole` clusters.
+- **Added** alpha support for in-proxy generation of service metrics using Prometheus.
+- **Added** alpha support for environmental metadata in Envoy node metadata.
+- **Added** alpha support for Proxy Metadata Exchange.
+- **Added** alpha support for the OpenCensus trace driver.
+- **Improved** the mesh dashboard to provide monitoring of Istio's configuration state.
+- **Improved** the Pilot dashboard to expose additional key metrics to more clearly identify errors.
+- **Removed** deprecated `Adapter` and `Template` custom resource definitions (CRDs).
+- **Deprecated** Mixer adapters. Please begin the transition of all extensions to the new extension model. Legacy Mixer integration support continues only for Istio 1.3 and 1.4.
+- **Deprecated** Deprecated HTTP API Spec. The support for producing API attributes will be be removed in Istio 1.4.
+
+## Policy
+
+- **Improved** rate limit enforcement to allow communication when the quota backend is unavailable.
+
+## Configuration Management
+
+- **Added** a configuration for Galley to use new configuration processing.
+- **Fixed** Galley to stop too many gRPC pings from closing connections.
+- **Improved** Galley to avoid control plane upgrade failures.
+
+## istioctl
+
+- **Added** `istioctl kube-uninject` to remove Istio configuration from a previously injected configuration.
+- **Added** `istioctl experimental describe pod` to produce the Istio configuration affecting a Pod.
+- **Added** `istioctl experimental manifest apply` to generate and apply an Istio installation.
+- **Added** `istioctl experimental manifest diff` to show the difference betweeen two manifests.
+- **Added** `istioctl experimental manifest generate` to generate an Istio installation manifest.
+- **Added** `istioctl experimental manifest migrate` to convert Istio's `values.yaml` file into a format that suits the Istio operator.
+- **Added** `istioctl experimental manifest versions` to list all supported Istio versions.
+- **Added** `istioctl experimental profile diff Show` to show the difference between two Istio configuration profiles.
+- **Added** `istioctl experimental profile dump` to show an Istio configuration profile.
+- **Added** `istioctl experimental profile list` to list all Istio configuration profiles.
+- **Added** `istioctl experimental add-to-mesh` to add virtual machines to an existing Istio service mesh.
```

Comment:
This is not correct.  It is not only about vms.  It is adding services from K8S or VM to the mesh.

```
$ istioctl x add-to-mesh
Add workloads into Istio service mesh

Usage:
  istioctl experimental add-to-mesh [flags]
  istioctl experimental add-to-mesh [command]

Aliases:
  add-to-mesh, add

Available Commands:
  external-service Add external service(eg:services running on VM) to Istio service mesh
  service          Add Service to Istio service mesh

```

Reply:
@linsun addressed.  PTAL.  If satisfied. pease click `Resolve conversation.`